packages/devtools_app/pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dependencies:
   http: ^1.1.0
   image: ^4.1.3
   intl: ^0.19.0
+  json_rpc_2: ^3.0.0
```

Comment:
do we need to bring this dependency in? Is this for the RpcException type, and is this the type that DTD throws?

Reply:
Yes, this is for the `RpcException` type thrown by DTD. This used to be a dev dependency.